src/storage/src/protocol/client.rs

```diff
@@ -490,6 +492,7 @@ mod tests {
         #![proptest_config(ProptestConfig::with_cases(32))]
 
         #[test]
+        #[ignore]
```

Comment:
are we keeping this in the merge?

Reply:
Yes, it's very broken and Alexander is looking for a solution here https://github.com/MaterializeInc/materialize/issues/14543